src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,75 @@
+
+1. Where to fetch or lazy-load agent code
+2. Where to send observation data
+
+Today, many ad-blockers and security VPNs block client requests to third-party domains including New Relic's. As the number of internet users opting into the use of ad-blockers increase, customers will continue to lose observability on those devices. One solution would be to proxy those requests though an unblocked middleman. For example, `yoursite.com` can send its visitor analytics through its `nrbeacon.yoursite.com` proxy which would not be blocked by anti-adware being that it's considered a first-party (sub)domain.
+
+<Callout variant="important">
+  This feature is currently available for those using the copy-and-paste _or_ NPM installation methods. There is currently no UI or NerdGraph configuration options available. We are continuing to work on improving access to these and other configuration options.
+</Callout>
+
+## How to use a proxy [#use]
+
+The agent utilizes two configuration properties when determining where to fetch agent code or send observability data.
+
+For example, using copy and paste:
+
+```js
+window.NREUM.init = {
+    ...<other init properties>,
+    proxy: {
+        assets: <URL to fetch agent chunks>,
+        beacon: <URL to send analytics>
+    }
+}
+```
+
+For NPM, similarly just pass the `init` object with a proxy as part of any other configurations:
+
+```js
+new BrowserAgent({ init: { proxy: <object similar to above> } })
+```
+
+`init.proxy.assets` is used verbatim in the copy-and-paste installation to fetch agent code.
+
+* For example, the v1.240.0 Pro+SPA loader will require `nr-spa.<hash>-1.240.min.js` file.
+* If this field is set to be `nrassets.abc.com` reverse proxy, provide this value as the exact URL, including the scheme and ending slash: `'https://nrassets.abc.com/'`.
+
+This configuration is not relevant to NPM installed agents. NPM installed agents will undergo the same bundling process as the host application and should result in agent assets in the output directory along side the host application. When the host application is deployed, those agent assets should be deployed as well resulting in agent code already loading from the same domain as the site.
+
+`init.proxy.beacon` is used by both installation to allow proxying of observability data through a different domain. For example, if this field is set to be `nrbeacon.abc.com`, all observability data will be sent to `https://nrbeacon.abc.com/<observability endpoint>` instead of what is configured automatically on the New Relic account and found in the `NREUM.info.beacon` configuration.
+
+<Callout variant="important">
+These configurations should follow a pattern of `<domain>[:<port>][/<path>]`. The protocol `https://` and ending slash **should be omitted**. The `port` and `path` are optional and will be respected.
+</Callout>
```

Comment:
This is where I think we have some confusion in the agent. The asset proxy value is expected to have the protocol and not end in a slash. The beacon proxy value is expected to not have the protocol and not end in a slash.

Reply:
Related comment on https://github.com/newrelic/docs-website/pull/14672#discussion_r1338578391